chidioguejiofor/maintenance-react-app
# #61050287/: create admin dashboard

#### What does this PR do?
- adds an AdminRoute to the app
- implements the Admin dashboard  
- improves coverage to 95.82%
- adds admin update features like approve, disapprove and resolve requests

#### Description of Task to be completed?
- admin should be able to update requests

#### How should this be manually tested?
- request admin credentials from the author
- login via `/admin/login
- you would see the admin dashboard

#### Any background context you want to provide?
- merging this would automatically update the Heroku app @ https://chidiebere-maintenance-react.herokuapp.com

#### What are the relevant pivotal tracker stories?
- 161050287

#### Screenshots (if appropriate)
<img width="1440" alt="screen shot 2018-10-09 at 12 33 52 am" src="https://user-images.githubusercontent.com/38565349/46638896-0c158700-cb5b-11e8-9483-f7d68fe2f4ed.png">
<img width="1440" alt="screen shot 2018-10-09 at 12 33 34 am" src="https://user-images.githubusercontent.com/38565349/46638897-0c158700-cb5b-11e8-90e7-1c5d497dea0f.png">
<img width="1440" alt="screen shot 2018-10-09 at 12 32 37 am" src="https://user-images.githubusercontent.com/38565349/46638898-0cae1d80-cb5b-11e8-9c0f-958991b10ba8.png">



#### Questions:
- none
